model/dd2d/pario_nc.f

```diff
@@ -778,8 +783,17 @@ subroutine stoprc(rc,rc_ok)
       integer :: rc,rc_ok
       integer :: mpi_err
 #ifndef SERIAL_MODE
+      call debug_msg("### Calling mpi_barrier")
+      call mpi_barrier(MPI_COMM_WORLD,mpi_err)
```

Comment:
Hm, something odd is going on because the non-root ranks hang at this barrier, too.